staging/src/k8s.io/apiserver/pkg/storage/errors.go

```diff
@@ -193,3 +193,21 @@ func IsTooLargeResourceVersion(err error) bool {
 	}
 	return errors.HasStatusCause(err, metav1.CauseTypeResourceVersionTooLarge)
 }
+
+// Simple storage errors with just a constant message.
+type simpleStorageError int
+
+const (
+	ErrResourceVersionSetOnCreate simpleStorageError = iota
+)
+
+var simpleStorageErrorText = map[simpleStorageError]string{
```

Comment:
That's way too overcomplicated to me - please don't create any maps or anything like that.

Please do the simplest possible thing.

Reply:
can't we just use `var ErrResourceVersionSetOnCreate = errors.New("resourceVersion should not be set on objects to be created")`? That seems simpler to me.

Reply:
Ok, changed to `var ErrResourceVersionSetOnCreate = errors.New("resourceVersion should not be set on objects to be created")`